frontend/app/views/fragments/global/header.scala.html

```diff
@@ -6,18 +6,44 @@
 @import configuration.Config
 @import model.SVG.Logos
 
+@if(!pageInfo.url.startsWith("/bundles")) {
```

Comment:
Check the possibility of extending the behaviour  built by @Ap0c (in the PR https://github.com/guardian/membership-frontend/pull/1368), maybe by creating a different cases of the header.